pkg/apis/k0s/v1beta1/images_test.go

```diff
@@ -106,28 +106,77 @@ func TestImagesRepoOverrideInConfiguration(t *testing.T) {
 }
 
 func TestOverrideFunction(t *testing.T) {
```

Comment:
I know you didn't write this line, but I think this should be TestOverrideRepository instead